scipy/_lib/_util.py

```diff
@@ -657,6 +657,33 @@ def __call__(self, x):
         return self.f(x, *self.args)
 
 
+class _ScalarFunctionWrapper:
+    """
+    Object to wrap scalar user function, allowing picklability
+    """
+    def __init__(self, f, args=None):
+        self.f = f
+        self.args = [] if args is None else args
+        self.nfev = 0
```

Comment:
`_ScalarFunctionWrapper` can't be relied on to keep track of number of function evaluations in a parallel context.

Reply:
(see comment in `numdiff`)